flink-python/pyflink/table/expressions.py

```diff
@@ -350,14 +350,19 @@ def temporal_overlaps(left_time_point,
 def date_format(timestamp, format) -> Expression:
     """
     Formats a timestamp as a string using a specified format.
-    The format must be compatible with MySQL's date formatting syntax as used by the
-    date_parse function.
```

Comment:
if this has been removed - do we not need to deprecate first before replacing?

Reply:
@davidradl , the comment I deleted is not accurate. 
MySQL's date formatting syntax follows its own date formatting syntax, not Java's `SimpleDateFormat` or `DateTimeFormatter`. MySQL is written in C++, not Java, and doesn't have much significance as a reference here.